packages/react-composites/src/composites/CallComposite/components/buttons/Microphone.tsx

```diff
@@ -46,14 +44,6 @@ export const Microphone = (props: {
     // page until the user successfully joins the call.
     microphoneButtonProps.checked = isLocalMicrophoneEnabled;
   }
-  const microphoneButtonStrings = _isInLobbyOrConnecting(callStatus)
```

Comment:
Can you link the original bug? it is unclear what scenario removing this code fixes

Reply:
@dmceachernmsft 
https://skype.visualstudio.com/SPOOL/_workitems/edit/3633982

The aria label was indicating we can't unmute/mute in the lobby however we can. We want to shift the aria label behavior to the default usage since we *can* mute/unmute in the lobby